internal/addrs/action.go

```diff
@@ -211,6 +211,7 @@ func (a AbsAction) UniqueKey() UniqueKey {
 // AbsActionInstance is an absolute address for an action instance under a
 // given module path.
 type AbsActionInstance struct {
+	referenceable
```

Comment:
are you sure this is what you want? generally we don't let people draw reference actions in other modules anyway so you can always just use the local reference directly?

Reply:
Yeah that's a good point 👍